dask_expr/io/io.py

```diff
@@ -129,6 +130,16 @@ def _combine_similar(self, root: Expr):
 class FusedIO(BlockwiseIO):
     _parameters = ["expr"]
 
+    @functools.cached_property
+    def _name(self):
+        return (
+            funcname(type(self.operand("expr"))).lower()
+            + "-"
+            + funcname(type(self)).lower()
```

Comment:
Maybe just `fused` rather than `fusedio` here?  (unless this is commonly subclassed)

Reply:
don't see a need for subclassing at the moment, I adjusted it